sys-apps/byld/byld-1.0.3.ebuild

```diff
@@ -1,23 +1,26 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-DESCRIPTION="build a Linux distribution on a single floppy"
+DESCRIPTION="Build a Linux distribution on a single floppy"
 HOMEPAGE="http://byld.sourceforge.net/"
 SRC_URI="mirror://sourceforge/byld/byld-${PV//./_}.tgz"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="x86"
-IUSE=""
-RESTRICT="strip" #252054
+
```

Comment:
```suggestion

```
`RESTRICT` is always part of the `LICENSE` block

Reply:
:+1: